charts/geoserver/templates/secrets.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
```

Comment:
lets use lower case username and password. And use existingSecret so it does not have to be specified here.

Reply:
need to encapsulate it with:
{{- if not .Values.auth.existingSecret }}

Reply:
It looks to me that geoserver office docker image doesn't have environment variable for admin and username but change them after logged in with default password. If so, we probably don't need this secret anymore. I will dig about this more

Reply:
secret has been removed because it is not needed for this one